### PR TITLE
Update community group hashtag in the Get involved` page

### PR DESCRIPTION
**Pull request by fasihahmad**
# Get Involved page updates

**Summary**: Community Group on display corrected to its proper hashtag

**Updated**:

1. `#Status-ambassadors -> #statusphere`
2. `#Status-Korean -> #status-korean`
3. New url scheme for public chat

### CODE REVIEW

**Comment by jinhojang6** [comment]
Nice PR @fasihahmad ! 

Speaking of which, I think we don't need to translate 
#statushpere, #status-korean, #status-spanish, #status-chinese, #status-philippines, and #philiosophers into other languages for the [Community Groups](https://status.im/get-involved/community-groups/) page.

Can you remove all the related keys ([example 1](https://github.com/status-im/status.im/blob/d6ce8d464458256d706c6b1b7a4df65d3914a93b/themes/navy/languages/en.yml#L464), [example 2](https://github.com/status-im/status.im/blob/d6ce8d464458256d706c6b1b7a4df65d3914a93b/themes/navy/languages/en.yml#L469)) above from [the 9 language files](https://github.com/status-im/status.im/tree/develop/themes/navy/languages)? 

And we should replace the removed keys such as [<%- __('site.get-involved.community.community-groups.account-group-1.link') %>](https://github.com/status-im/status.im/blob/d6ce8d464458256d706c6b1b7a4df65d3914a93b/themes/navy/layout/community.ejs#L45) with the copy (e.g., #statusphere). To do this, 6 lines should be updated in [community.ejs](https://github.com/status-im/status.im/blob/develop/themes/navy/layout/community.ejs).

What do you think of it?

**Comment by fasihahmad** [comment]
> [community.ejs](https://github.com/status-im/status.im/blob/develop/themes/navy/layout/community.ejs).

Absolutely right @jinhojang6 we don't need to translate.

You should remove `link` key from different community account groups which will help us
to protect these issues to be raised again while translating into more languages in future.

But I have a little concern, as suggested if we will update `community.ejs` directly with
data then there will be no separation with data and template for maintaining in future.

I saw a file `source/_data/channels.yml` but problem with this file is, it keeps channel
based on language. 

So currently I am doing exactly what you have suggested.

**Comment by jinhojang6** [comment]
@fasihahmad I agree with you. The data should be separated from the template :)

I added Jakub as a reviewer for double-checking.

File: themes/navy/layout/community.ejs [hunk 1]
```diff
@@ -42,7 +42,7 @@
 				<div class="col-lg-6 col-xl-4">
 					<div class="account-creation">
 						<%- image_tag("/img/status-ambassadors.svg") %>
-						<a href="https://join.status.im/chat/public/statusphere" target="_blank"><%- __('site.get-involved.community.community-groups.account-group-1.link') %></a>
+						<a href="https://join.status.im/chat/public/statusphere" target="_blank">#statusphere</a>
```
Comment (jakubgs) [inline themes/navy/layout/community.ejs:45]: These are old links. New links use the format https://join.status.im/statusphere for a public chat.

Comment (fasihahmad) [reply themes/navy/layout/community.ejs]: Updated new link @jakubgs

**Pull request merged**